### ISSUE TITLE
--grouped truncates long build output

### ISSUE THREAD

**Issue by ecraig12345** [microsoft/lage#114]
With the `--grouped` option and a task with very long output, some of the output is getting truncated (at least on errors). This is a problem because it can make determining the source of an error impossible--2 examples in [this build](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=154603&view=logs&j=a8a82d6e-f5e1-5356-31e4-5989f531914b&t=7435ac5c-7c8f-5a6b-afd7-ffd5f69a92ea). I've disabled grouped builds in Fluent UI for now (https://github.com/microsoft/fluentui/pull/16019) so that we can actually debug these issues, but I'd like to re-enable the option since it's a better experience in general.

**Comment by kenotron** [comment]
The problem also with --grouped is that it buffers... so if things are timed out, the log would actually never be written. I think if the log is supposed to be long, maybe turning it off is good enough? What do you think?

**Comment by ecraig12345** [comment]
Yeah turning it off is how we've worked around the issue so far. Some other ideas:
- Can we make lage detect when it's about to be killed and flush the logs?
- Maybe add a way to turn off grouping per project? (though for occasional weird issues or PRs that introduce new hangs, that wouldn't help)
